include/gnc_constants.hpp

```diff
@@ -70,15 +70,11 @@ constexpr static lin::Vector3d earth_precession_rate = 1.0e-11L * lin::Vector3d(
 
 /** Approximate precssion rate of Earth's Axis in radians per second (float
  *  version). */
-constexpr static lin::Vector3f earth_precession_rate_f = {
-  static_cast<float>(earth_precession_rate(0)),
-  static_cast<float>(earth_precession_rate(1)),
-  static_cast<float>(earth_precession_rate(2))
-};
+constexpr static lin::Vector3f earth_precession_rate_f = earth_precession_rate;
```

Comment:
Just to point it out, this is an example of how we'll be able to take advantage of implicit casts.